addons/l10n_es/models/account_move.py

```diff
@@ -0,0 +1,16 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, fields, api, _
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_es_is_simplified = fields.Boolean("Is Simplified",
+                                           compute="_compute_l10n_es_is_simplified", readonly=False, store=True)
+
+    @api.depends('partner_id')
+    def _compute_l10n_es_is_simplified(self):
+        simplified_partner = self.env.ref('l10n_es.partner_simplified', raise_if_not_found=False)
+        if simplified_partner:
+            for move in self:
+                move.l10n_es_is_simplified = (move.partner_id == simplified_partner)
```

Comment:
In the pos app you can choose a normal partner and still have a simplified invoice. The usecase is that you might want to assign a partner  ( for keeping track of rewards for example ) but you don't want to invoice this partner; only that we automatically invoice all orders so we'll end up creating a regular invoice for this partner ( because he is not the `simplified_partner` )
What do you think about this?

Reply:
That is exactly why the field was made like this: if there is no invoice to be created from the PoS and you have another partner than the simplified, you should set it to true explicitly.  (you can also set it with the simplified) @vlst-odoo

Reply:
ok, that makes sense